man/systemd-repart.xml

```diff
@@ -443,9 +443,13 @@
       <varlistentry>
         <term><option>--copy-from=</option><arg>IMAGE</arg></term>
 
-        <listitem><para>Instructs <command>systemd-repart</command> to copy the partitions from the given
-        image. The partitions from the given image are synthesized into partition definitions that are
-        parsed before the partition definition files.</para></listitem>
+        <listitem><para>Instructs <command>systemd-repart</command> to synthesize partition definitions from
+        the partition table in the given image. The generated definitions will copy the partitions into the
+        destination partition table. The copied partitions will have the same size, metadata and contents but
+        might have a different partition number and might be located at a different offset in the destination
+        partition table. These definitions can be combined with partition definitions read from regular
+        partition definition files. The synthesized definitions take precedence over the definitions read
+        from partition definition files.</para></listitem>
```

Comment:
hmm, why do they take precedence? that is surprising, no?

Reply:
We could switch the order. But I was thinking about using --copy-from to copy my image from USB to disk and then having a root partition definition in /usr/lib/repart.d that is also applied. The expected end result here would be that the root partition gets appended to the stuff from --copy-from. It'd be weird to have the root partition as the first partition.

Reply:
maybe it would make sense to give users control over the ordering. i.e. insert these synthetic definitions under a name 30-synthesized-1.conf, 32-synthesized.conf, 34-synthesized.conf and so on. That way, people can put their manual definitions before, after or even between these entries if they like.